static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -70,107 +67,90 @@ function GroupRelatedIssues({params}: Props) {
   );
 
   return (
-    <Layout.Body>
```

Comment:
Using hide whitespaces and split makes reviewing this PR easier
<img width="205" alt="image" src="https://github.com/getsentry/sentry/assets/44410/023f1922-bf7b-4828-a1ad-80c763e2ddf4">